src/legacy/ui/public/vis/editors/default/components/agg.tsx

```diff
@@ -98,9 +111,6 @@ function DefaultEditorAgg({
     }
   }, [lastParentPipelineAggTitle, isLastBucket, agg.type]);
 
-  // A description of the aggregation, for displaying in the collapsed agg header
-  const aggDescription = agg.type && agg.type.makeLabel ? agg.type.makeLabel(agg) : '';
```

Comment:
Couldn't we just use `safeMakeLabel` for this purpose?

Reply:
No (without changing of `safeMakeLabel`), because `safeMakeLabel` invokes `makeLabel` of `agg` and doesn't pass any param, but we need to call `makeLabel` of `agg.type` and pass `agg` as param.